Cockroach crashes with [runtime error index out of range]

Just quickly dropping this here, I will supply more details later.

Crash message:

```
CockroachDB node starting at 2017-07-26 13:28:00.553866934 +0000 UTC
build:      CCL v1.0.2 @ 2017/06/15 12:58:59 (go1.8.3)
admin:      http://roach1:8080
sql:        postgresql://root@roach1:26257?sslmode=disable
logs:       /cockroach/cockroach-data/logs
store[0]:   path=/cockroach/cockroach-data
status:     restarted pre-existing node
clusterID:  c701695a-d8d3-43d9-be51-27047794b409
nodeID:     1
*
* ERROR: [n1] a panic has occurred!
*
*
* ERROR: [n1] Reported as error b121946441924ceb85beb1c99cab99d1
*
runtime error: index out of range

goroutine 85996 [running]:
runtime/debug.Stack(0x7fb631e164d0, 0xc420274000, 0x192bf80)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x79
github.com/cockroachdb/cockroach/pkg/util/log.ReportPanic(0x7fb631e164d0, 0xc420274000, 0x192bf80, 0x2747ee0, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/crash_reporting.go:115 +0x32b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Recover(0xc4206ab680, 0x7fb631e164d0, 0xc420274000)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:199 +0x99
panic(0x192bf80, 0x2747ee0)
	/usr/local/go/src/runtime/panic.go:489 +0x2cf
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).serve.func1(0xc42255ee00, 0x7fb631e164d0, 0xc42b184b40)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:353 +0x74
panic(0x192bf80, 0x2747ee0)
	/usr/local/go/src/runtime/panic.go:489 +0x2cf
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).handleBind(0xc42255ee00, 0x7fb631e166f0, 0xc42b2c9200, 0xc42255ee28, 0x176014, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:741 +0x61d
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).serve(0xc42255ee00, 0x7fb631e164d0, 0xc42b184b40, 0xc420e8c360, 0x5400, 0xc42043a310, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:451 +0x9cd
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc42043a1c0, 0x7fb631e166f0, 0xc42b2c9180, 0x278f280, 0xc420c92d20, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:421 +0x926
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func9.1(0x278f280, 0xc420c92d20)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:615 +0x149
github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1(0xc4206ab680, 0x7fb631e164d0, 0xc420274000, 0xc42000e038, 0x278f280, 0xc420c92d20, 0xc4202f4510)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:142 +0xa9
created by github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:144 +0x267
```

```
grep -F '[config]' cockroach.log 
I170726 13:27:54.020674 1 util/log/clog.go:1011  [config] file created at: 2017/07/26 13:27:54
I170726 13:27:54.020674 1 util/log/clog.go:1011  [config] running on machine: roach1
I170726 13:27:54.020674 1 util/log/clog.go:1011  [config] binary: CockroachDB CCL v1.0.2 (linux amd64, built 2017/06/15 12:58:59, go1.8.3)
I170726 13:27:54.020674 1 util/log/clog.go:1011  [config] arguments: [/cockroach/cockroach start --insecure]
```

Ignore the 1.0.2 I just upgraded and tested it again, still the same problem. I will provide  details about the query executed soon. All I know currently is that it is related to the number of parameters, as if I reduce them to a smaller amount everything works fine with the same queries.

As said deeper details are provided later.

A complete log from start to crash:

```
CockroachDB node starting at 2017-07-26 17:07:11.474414372 +0000 UTC
build:      CCL v1.0.3 @ 2017/07/06 17:46:06 (go1.8.3)
admin:      http://roach1:8080
sql:        postgresql://root@roach1:26257?sslmode=disable
logs:       /cockroach/cockroach-data/logs
store[0]:   path=/cockroach/cockroach-data
status:     restarted pre-existing node
clusterID:  c701695a-d8d3-43d9-be51-27047794b409
nodeID:     1
W170726 17:07:11.487646 187 storage/node_liveness.go:291  [n1,hb] slow heartbeat took 12.3s
I170726 17:07:11.487661 187 storage/node_liveness.go:250  [n1,hb] heartbeat failed on epoch increment; retrying
I170726 17:07:11.501574 187 storage/node_liveness.go:250  [n1,hb] heartbeat failed on epoch increment; retrying
I170726 17:07:12.170455 3348 gossip/server.go:234  [n1] received initial cluster-verification connection from {tcp roach3:26269}
I170726 17:07:13.100701 3401 storage/stores.go:312  [n1] wrote 3 node addresses to persistent storage
I170726 17:07:13.540487 3474 storage/raft_transport.go:436  [n1] raft transport stream to node 3 established
I170726 17:07:13.785835 74 storage/replica_proposal.go:415  [n1,s1,r852/1:/Table/73{2-3}] new range lease repl=(n1,s1):1 start=1500159988.869440261,0 epo=27 pro=1501088833.769183063,0 following repl=(n1,s1):1 start=1500159988.869440261,0 epo=21 pro=1500159988.869442755,0
I170726 17:07:13.990024 102 storage/replica_proposal.go:415  [n1,s1,r923/1:/Table/78{4-5}] new range lease repl=(n1,s1):1 start=1499859090.720775631,0 epo=27 pro=1501088833.969461480,0 following repl=(n1,s1):1 start=1499859090.720775631,0 epo=17 pro=1500102987.677486017,0
I170726 17:07:15.183255 74 storage/replica_proposal.go:415  [n1,s1,r919/1:/Table/78{0-1}] new range lease repl=(n1,s1):1 start=1501088835.171842365,0 epo=27 pro=1501088835.171850052,0 following repl=(n2,s2):2 start=1500159989.773087023,0 epo=18 pro=1500159989.773088226,0
I170726 17:07:15.385098 110 storage/replica_proposal.go:415  [n1,s1,r541/1:/Table/5{19-20}] new range lease repl=(n1,s1):1 start=1501088835.372309901,0 epo=27 pro=1501088835.372317676,0 following repl=(n2,s2):2 start=1500159989.148783324,0 epo=18 pro=1500159989.148785287,0
I170726 17:07:15.995649 45 storage/replica_proposal.go:415  [n1,s1,r727/1:/Table/64{3-4}] new range lease repl=(n1,s1):1 start=1500117561.354979922,0 epo=27 pro=1501088835.973396253,0 following repl=(n1,s1):1 start=1500117561.354979922,0 epo=21 pro=1500117561.354984252,0
I170726 17:07:16.392843 98 storage/replica_proposal.go:415  [n1,s1,r934/1:/Table/79{5-6}] new range lease repl=(n1,s1):1 start=1499859090.720775631,0 epo=27 pro=1501088836.374178931,0 following repl=(n1,s1):1 start=1499859090.720775631,0 epo=21 pro=1500150156.089573101,0
I170726 17:07:16.594946 107 storage/replica_proposal.go:415  [n1,s1,r408/1:/Table/41{8-9}] new range lease repl=(n1,s1):1 start=1500153793.992467234,0 epo=27 pro=1501088836.574617136,0 following repl=(n1,s1):1 start=1500153793.992467234,0 epo=21 pro=1500153793.992560428,0
I170726 17:07:16.830243 120 storage/replica_proposal.go:415  [n1,s1,r443/1:/Table/42{1-2}] new range lease repl=(n1,s1):1 start=1500146012.330342390,0 epo=27 pro=1501088836.774969844,0 following repl=(n1,s1):1 start=1500146012.330342390,0 epo=21 pro=1500146012.330372544,0
I170726 17:07:16.986639 82 storage/replica_proposal.go:415  [n1,s1,r266/1:/Table/29{4-5}] new range lease repl=(n1,s1):1 start=1501088836.975359602,0 epo=27 pro=1501088836.975364884,0 following repl=(n2,s2):2 start=1500159988.869800407,0 epo=18 pro=1500159988.869802054,0
I170726 17:07:17.392170 92 storage/replica_proposal.go:415  [n1,s1,r697/1:/Table/62{3-4}] new range lease repl=(n1,s1):1 start=1501088837.376160606,0 epo=27 pro=1501088837.376165626,0 following repl=(n3,s3):3 start=1500159988.805810849,0 epo=18 pro=1500159988.805811916,0
I170726 17:07:17.604238 77 storage/replica_proposal.go:415  [n1,s1,r859/1:/Table/7{39-40}] new range lease repl=(n1,s1):1 start=1499859090.720775631,0 epo=27 pro=1501088837.576474450,0 following repl=(n1,s1):1 start=1499859090.720775631,0 epo=21 pro=1500143988.905914561,0
I170726 17:07:18.009850 69 storage/replica_proposal.go:415  [n1,s1,r329/1:/Table/35{7-8}] new range lease repl=(n1,s1):1 start=1501088837.977189473,0 epo=27 pro=1501088837.977194029,0 following repl=(n2,s2):2 start=1500159989.569577222,0 epo=18 pro=1500159989.569579061,0
I170726 17:07:18.596873 111 storage/replica_proposal.go:415  [n1,s1,r700/1:/Table/62{6-7}] new range lease repl=(n1,s1):1 start=1500159988.758051132,0 epo=27 pro=1501088838.578297952,0 following repl=(n1,s1):1 start=1500159988.758051132,0 epo=21 pro=1500159988.758066608,0
I170726 17:07:18.789370 89 storage/replica_proposal.go:415  [n1,s1,r568/1:/Table/54{6-7}] new range lease repl=(n1,s1):1 start=1501088838.778921933,0 epo=27 pro=1501088838.778930667,0 following repl=(n2,s2):2 start=1500159989.569509043,0 epo=18 pro=1500159989.569510295,0
I170726 17:07:19.140420 148 server/status/runtime.go:225  [n1] runtime stats: 232 MiB RSS, 241 goroutines, 42 MiB/9.4 MiB/65 MiB GO alloc/idle/total, 76 MiB/90 MiB CGO alloc/total, 1866.72cgo/sec, 0.21/0.05 %(u/s)time, 0.00 %gc (15x)
W170726 17:07:19.140812 146 server/node.go:719  [n1] [n1,s1]: unable to compute metrics: [n1,s1]: system config not yet available
I170726 17:07:19.189237 71 storage/replica_proposal.go:415  [n1,s1,r42/1:/Table/8{1-2}] new range lease repl=(n1,s1):1 start=1501088839.179524674,0 epo=27 pro=1501088839.179530592,0 following repl=(n2,s2):2 start=1500159989.787764373,0 epo=18 pro=1500159989.787766910,0
I170726 17:07:19.427911 93 storage/replica_proposal.go:415  [n1,s1,r5/1:/Table/{0-11}] new range lease repl=(n1,s1):1 start=1501088541.975369105,0 epo=25 pro=1501088541.975370880,0 following repl=(n2,s2):2 start=1501075678.961793739,0 epo=20 pro=1501075680.159197679,0
I170726 17:07:19.433545 45 storage/replica_proposal.go:415  [n1,s1,r842/1:/Table/72{2-3}] new range lease repl=(n1,s1):1 start=1501088839.379810630,0 epo=27 pro=1501088839.379815501,0 following repl=(n3,s3):3 start=1500159988.944030119,0 epo=18 pro=1500159988.944031421,0
I170726 17:07:19.520762 120 storage/replica_proposal.go:415  [n1,s1,r7/1:/Table/1{2-3}] new range lease repl=(n1,s1):1 start=1501088541.975637669,0 epo=25 pro=1501088541.975639432,0 following repl=(n2,s2):2 start=1501075680.799868002,0 epo=20 pro=1501075680.799871589,0
I170726 17:07:19.548677 46 storage/replica_proposal.go:415  [n1,s1,r9/1:/Table/1{4-5}] new range lease repl=(n1,s1):1 start=1500117516.857849051,0 epo=27 pro=1501088839.504115382,0 following repl=(n1,s1):1 start=1500117516.857849051,0 epo=24 pro=1501076177.042616267,0
I170726 17:07:19.559047 3941 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.559164 3941 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.559634 3941 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.559985 3941 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:19.560328 3941 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.560489 3839 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.560604 3839 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.560675 3839 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.560816 3839 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.561255 3839 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.561777 3839 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 520 KiB (+20480)
I170726 17:07:19.565505 3938 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.565628 3938 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.565825 3938 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.566256 3938 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.567319 3941 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.569816 3938 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.577860 3839 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.586072 3938 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.612890 3941 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.620078 3839 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.620149 3839 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:19.631018 94 storage/replica_proposal.go:415  [n1,s1,r105/1:/Table/13{4-5}] new range lease repl=(n1,s1):1 start=1500027019.655551852,0 epo=27 pro=1501088839.582499300,0 following repl=(n1,s1):1 start=1500027019.655551852,0 epo=21 pro=1500134377.640556621,0
I170726 17:07:19.638174 4002 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.638297 4002 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.638492 4002 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.638816 4002 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.638952 4002 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.660060 4002 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.666033 4038 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.666106 4038 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.666240 4038 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.666535 4038 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.666672 4038 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.669104 4038 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.674905 4002 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.681627 4038 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.713588 4165 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:19.720675 4165 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.720785 4165 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.720851 4165 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:19.720984 4165 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.721482 4165 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.721690 3544 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.724219 4165 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.724916 3544 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 16 KiB (+225)
I170726 17:07:19.725165 3544 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 33 KiB (+1080)
I170726 17:07:19.726008 3544 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 64 KiB (+219)
I170726 17:07:19.727791 3544 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 132 KiB (+4320)
I170726 17:07:19.729010 4165 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.731708 3210 sql/event_log.go:101  [n1] Event: "node_restart", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:roach1:26257} Attrs: Locality:} ClusterID:c701695a-d8d3-43d9-be51-27047794b409 StartedAt:1501088819139793181 LastUp:1501088306724364191}
I170726 17:07:19.732698 3544 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 256 KiB (+239)
I170726 17:07:19.733784 4165 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.735691 3544 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 520 KiB (+10240)
I170726 17:07:19.742048 4190 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.742185 4190 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.742365 4190 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.742691 4190 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.742768 4190 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.744992 4190 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.749125 3998 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.749172 3998 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:19.749262 3998 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.749461 3998 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.754011 3998 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.754406 3998 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 270 KiB (+20480)
I170726 17:07:19.760036 3998 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.767818 4190 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.768005 4231 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.768039 4231 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:19.768118 4231 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.768285 4231 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.768768 4231 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.768910 4218 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.773735 4231 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.777747 3998 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.780111 4218 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.780219 4218 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.780494 4218 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.780782 4218 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.780921 4218 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.783790 4218 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.785434 4231 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.793322 4356 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.793448 4356 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.793627 4356 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.793649 4356 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.794102 4356 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.795096 4218 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.796954 4356 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.808454 4356 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.831496 4460 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.831604 4460 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.831787 4460 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.832296 4460 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.832335 4460 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.838262 4460 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.856353 4497 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.856461 4497 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.856632 4497 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.857018 4497 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.857254 4497 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+20480)
I170726 17:07:19.858005 4460 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.862150 4497 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.871453 4626 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:19.878029 4510 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.878145 4510 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.878334 4510 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.878693 4510 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.878797 4510 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.881299 4510 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.884101 4626 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.884211 4626 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.884378 4626 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.884565 4497 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.884678 4626 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.884824 4626 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.891397 4626 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.907630 4510 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.910799 4626 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.911545 4633 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.911650 4633 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.911776 4633 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.911861 4633 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.912167 4633 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.916062 4633 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.931754 4633 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.945145 4759 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.945253 4759 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.945450 4759 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.945623 4759 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.945906 4759 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.953054 4759 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.964801 4759 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.973381 4852 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.973498 4852 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.973681 4852 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.974091 4852 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.974126 4852 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.979669 4852 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:19.993837 4852 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:19.996529 4915 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:19.996627 4915 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:19.996808 4915 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:19.997127 4915 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:19.997164 4915 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:19.999754 4915 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:20.009636 75 storage/replica_proposal.go:415  [n1,s1,r876/1:/Table/74{8-9}] new range lease repl=(n1,s1):1 start=1501088839.983694906,0 epo=27 pro=1501088839.983729057,0 following repl=(n3,s3):3 start=1500159988.723353663,0 epo=18 pro=1500159988.723355050,0
I170726 17:07:20.012311 4954 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:20.012432 4954 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:20.013609 4954 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:20.014029 4954 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:20.014092 4954 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:20.015081 4915 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:20.016589 4954 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:20.025577 4895 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:20.025710 4895 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:20.025907 4895 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:20.026166 4895 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:20.026343 4895 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:20.033682 4954 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:20.037139 4895 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:20.047668 4895 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:20.053427 4988 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 16 KiB (+216)
I170726 17:07:20.053548 4988 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 32 KiB (+180)
I170726 17:07:20.053616 4988 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:20.053788 4988 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 64 KiB (+218)
I170726 17:07:20.054246 4988 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 128 KiB (+181)
I170726 17:07:20.059310 4988 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 256 KiB (+167)
I170726 17:07:20.067249 4988 sql/mon/mem_usage.go:520  [n1] txn: memory usage increases to 17 KiB (+2128)
I170726 17:07:20.602561 83 storage/replica_proposal.go:415  [n1,s1,r482/1:/Table/46{0-1}] new range lease repl=(n1,s1):1 start=1501088840.584943743,0 epo=27 pro=1501088840.584951243,0 following repl=(n2,s2):2 start=1500159989.779303954,0 epo=18 pro=1500159989.779560854,0
I170726 17:07:20.800509 68 storage/replica_proposal.go:415  [n1,s1,r531/1:/Table/5{09-10}] new range lease repl=(n1,s1):1 start=1501088840.785305993,0 epo=27 pro=1501088840.785310374,0 following repl=(n3,s3):3 start=1500159989.075163661,0 epo=18 pro=1500159989.075167189,0
I170726 17:07:21.204055 103 storage/replica_proposal.go:415  [n1,s1,r566/1:/Table/54{4-5}] new range lease repl=(n1,s1):1 start=1499030127.840840387,0 epo=27 pro=1501088841.186050968,0 following repl=(n1,s1):1 start=1499030127.840840387,0 epo=21 pro=1500143584.220986339,0
I170726 17:07:22.202364 66 storage/replica_proposal.go:415  [n1,s1,r89/1:/Table/11{8-9}] new range lease repl=(n1,s1):1 start=1500132556.558315765,0 epo=27 pro=1501088842.188233870,0 following repl=(n1,s1):1 start=1500132556.558315765,0 epo=21 pro=1500132556.558358298,0
I170726 17:07:22.405418 81 storage/replica_proposal.go:415  [n1,s1,r486/1:/Table/46{4-5}] new range lease repl=(n1,s1):1 start=1500159989.569401613,0 epo=27 pro=1501088842.388793908,0 following repl=(n1,s1):1 start=1500159989.569401613,0 epo=21 pro=1500159989.569403054,0
I170726 17:07:22.609770 115 storage/replica_proposal.go:415  [n1,s1,r925/1:/Table/78{6-7}] new range lease repl=(n1,s1):1 start=1501023809.980497274,0 epo=27 pro=1501088842.589665874,0 following repl=(n1,s1):1 start=1501023809.980497274,0 epo=22 pro=1501023809.980499184,0
I170726 17:07:22.808478 42 storage/replica_proposal.go:415  [n1,s1,r508/1:/Table/48{6-7}] new range lease repl=(n1,s1):1 start=1500159988.820314576,0 epo=27 pro=1501088842.790030360,0 following repl=(n1,s1):1 start=1500159988.820314576,0 epo=21 pro=1500159988.820316125,0
I170726 17:07:23.011264 74 storage/replica_proposal.go:415  [n1,s1,r151/1:/Table/1{79-80}] new range lease repl=(n1,s1):1 start=1500138525.660425709,0 epo=27 pro=1501088842.991216209,0 following repl=(n1,s1):1 start=1500138525.660425709,0 epo=21 pro=1500138525.660496165,0
I170726 17:07:23.207575 87 storage/replica_proposal.go:415  [n1,s1,r924/1:/Table/78{5-6}] new range lease repl=(n1,s1):1 start=1500159989.736874213,0 epo=27 pro=1501088843.191416042,0 following repl=(n1,s1):1 start=1500159989.736874213,0 epo=21 pro=1500159989.736876023,0
I170726 17:07:23.407942 111 storage/replica_proposal.go:415  [n1,s1,r987/1:/Table/83{5-6}] new range lease repl=(n1,s1):1 start=1500159988.791021934,0 epo=27 pro=1501088843.391814793,0 following repl=(n1,s1):1 start=1500159988.791021934,0 epo=21 pro=1500159988.791023488,0
I170726 17:07:23.809695 115 storage/replica_proposal.go:415  [n1,s1,r344/1:/Table/37{2-3}] new range lease repl=(n1,s1):1 start=1500125575.746584261,0 epo=27 pro=1501088843.792773675,0 following repl=(n1,s1):1 start=1500125575.746584261,0 epo=21 pro=1500125575.746666320,0
I170726 17:07:23.963393 117 storage/replica_proposal.go:415  [n1,s1,r6/1:/Table/1{1-2}] new range lease repl=(n1,s1):1 start=1501088843.952197622,0 epo=27 pro=1501088843.952200538,0 following repl=(n3,s3):3 start=1501076088.965546300,0 epo=23 pro=1501076088.965550994,0
I170726 17:07:24.258776 77 storage/replica_proposal.go:415  [n1,s1,r841/1:/Table/72{1-2}] new range lease repl=(n1,s1):1 start=1499859090.720775631,0 epo=27 pro=1501088844.193855186,0 following repl=(n1,s1):1 start=1499859090.720775631,0 epo=22 pro=1501015240.142461791,0
I170726 17:07:24.616343 94 storage/replica_proposal.go:415  [n1,s1,r484/1:/Table/46{2-3}] new range lease repl=(n1,s1):1 start=1500159988.790820443,0 epo=27 pro=1501088844.594726205,0 following repl=(n1,s1):1 start=1500159988.790820443,0 epo=21 pro=1500159988.790822282,0
I170726 17:07:24.818167 79 storage/replica_proposal.go:415  [n1,s1,r723/1:/Table/6{39-40}] new range lease repl=(n1,s1):1 start=1500147629.768065341,0 epo=27 pro=1501088844.795321012,0 following repl=(n1,s1):1 start=1500147629.768065341,0 epo=21 pro=1500147629.768114762,0
I170726 17:07:24.944874 25 storage/replica_raftstorage.go:415  [raftsnapshot,n1,s1,r641/1:/System/tsd/cr.node.sql.…] generated Raft snapshot 92d354bb at index 42981
I170726 17:07:25.206249 93 storage/replica_proposal.go:415  [n1,s1,r150/1:/Table/17{8-9}] new range lease repl=(n1,s1):1 start=1501088845.196153590,0 epo=27 pro=1501088845.196159266,0 following repl=(n3,s3):3 start=1500159988.956320361,0 epo=18 pro=1500159988.956322533,0
I170726 17:07:27.772049 25 storage/store.go:3344  [raftsnapshot,n1,s1,r641/1:/System/tsd/cr.node.sql.…] streamed snapshot to (n3,s3):3: kv pairs: 4815, log entries: 3, rate-limit: 8.0 MiB/sec, 2827ms
I170726 17:07:29.140853 148 server/status/runtime.go:225  [n1] runtime stats: 252 MiB RSS, 190 goroutines, 26 MiB/29 MiB/71 MiB GO alloc/idle/total, 100 MiB/117 MiB CGO alloc/total, 1318.76cgo/sec, 0.18/0.04 %(u/s)time, 0.00 %gc (16x)
I170726 17:07:29.912220 5538 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 16 KiB (+332)
I170726 17:07:29.912414 5538 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 32 KiB (+1080)
I170726 17:07:29.912805 5538 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 64 KiB (+223)
I170726 17:07:29.913462 5538 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:29.913600 5538 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 128 KiB (+289)
I170726 17:07:29.915113 5538 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:29.915214 5538 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 256 KiB (+225)
I170726 17:07:30.316014 112 storage/replica_proposal.go:415  [n1,s1,r603/1:/Table/58{0-1}] new range lease repl=(n1,s1):1 start=1501088845.998117627,0 epo=27 pro=1501088845.998146852,0 following repl=(n2,s2):2 start=1500159988.871077314,0 epo=18 pro=1500159988.871078411,0
I170726 17:07:30.383604 109 storage/replica_proposal.go:415  [n1,s1,r1015/1:/Table/86{3-4}] new range lease repl=(n1,s1):1 start=1501088847.401819651,0 epo=27 pro=1501088847.401873053,0 following repl=(n3,s3):3 start=1501083003.509470843,0 epo=23 pro=1501083003.509473216,0
I170726 17:07:35.468508 59 gossip/gossip.go:1210  [n1] node has connected to cluster via gossip
I170726 17:07:35.468969 59 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170726 17:07:35.810745 5328 sql/mon/mem_usage.go:520  [client=172.22.0.1:50656,user=root,n1] session: memory usage increases to 379 KiB (+387913)
I170726 17:07:35.918242 5328 sql/mon/mem_usage.go:520  [client=172.22.0.1:50656,user=root,n1] txn: memory usage increases to 754 KiB (+771456)
I170726 17:07:35.919160 5328 sql/mon/mem_usage.go:520  [client=172.22.0.1:50656,user=root,n1] txn: memory usage increases to 1.0 MiB (+162)
I170726 17:07:35.960752 90 storage/replica_proposal.go:415  [n1,s1,r1070/1:/{Table/873-Max}] new range lease repl=(n1,s1):1 start=1499859090.720775631,0 epo=27 pro=1501088855.942465822,0 following repl=(n1,s1):1 start=1499859090.720775631,0 epo=24 pro=1501082947.768632561,0
I170726 17:07:38.882556 5625 sql/mon/mem_usage.go:520  [client=172.22.0.1:50656,user=root,n1] session: memory usage increases to 521 KiB (+11760)
I170726 17:07:38.889344 5625 sql/mon/mem_usage.go:520  [client=172.22.0.1:50656,user=root,n1] session: memory usage increases to 1.0 MiB (+52920)
I170726 17:07:39.140470 148 server/status/runtime.go:225  [n1] runtime stats: 250 MiB RSS, 188 goroutines, 70 MiB/880 KiB/87 MiB GO alloc/idle/total, 101 MiB/126 MiB CGO alloc/total, 5913.66cgo/sec, 0.16/0.02 %(u/s)time, 0.00 %gc (6x)
I170726 17:07:40.008106 5553 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 16 KiB (+332)
I170726 17:07:40.008305 5553 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 32 KiB (+1080)
I170726 17:07:40.008730 5553 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 64 KiB (+223)
I170726 17:07:40.009393 5553 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 130 KiB (+10240)
I170726 17:07:40.009538 5553 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 128 KiB (+289)
I170726 17:07:40.011085 5553 sql/mon/mem_usage.go:520  [n1] admin: memory usage increases to 260 KiB (+10240)
I170726 17:07:40.011215 5553 sql/mon/mem_usage.go:520  [n1] session: memory usage increases to 256 KiB (+225)
I170726 17:07:41.673575 5328 sql/mon/mem_usage.go:520  [client=172.22.0.1:50656,user=root,n1] session: memory usage increases to 1004 KiB (+640233)
E170726 17:07:41.682915 5328 util/log/crash_reporting.go:82  [n1] a panic has occurred!
E170726 17:07:42.295638 5328 util/log/crash_reporting.go:174  [n1] Reported as error a9ba75444a4b47678ad5bf32cb83530b
panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range

goroutine 5328 [running]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Recover(0xc4204b6320, 0x7f2465bdc9c0, 0xc4204fc5a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:200 +0xb1
panic(0x192d620, 0x2749ee0)
	/usr/local/go/src/runtime/panic.go:489 +0x2cf
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).serve.func1(0xc423143200, 0x7f2465bdc9c0, 0xc421f5cb10)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:353 +0x74
panic(0x192d620, 0x2749ee0)
	/usr/local/go/src/runtime/panic.go:489 +0x2cf
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).handleBind(0xc423143200, 0x7f2465bdcbe0, 0xc42245ed00, 0xc423143228, 0x176014, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:741 +0x61d
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).serve(0xc423143200, 0x7f2465bdc9c0, 0xc421f5cb10, 0xc420cd7170, 0x5400, 0xc42066c310, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:451 +0x9cd
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc42066c1c0, 0x7f2465bdcbe0, 0xc42245ec80, 0x2791280, 0xc420498780, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:421 +0x926
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func9.1(0x2791280, 0xc420498780)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:615 +0x149
github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1(0xc4204b6320, 0x7f2465bdc9c0, 0xc4204fc5a0, 0xc4203b8020, 0x2791280, 0xc420498780, 0xc420ef3a40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:142 +0xa9
created by github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:144 +0x267
```

I am away for some hours, maybe it is getting late or even tomorrow. Will provide you an executable reproduction as soon as I get to it, unless you already know what the failure is :)

Hear ya later.

--- a/pkg/sql/pgwire/pgwire_test.go
+++ b/pkg/sql/pgwire/pgwire_test.go
@@ -1665,3 +1665,35 @@ func TestPGWireResultChange(t *testing.T) {
 		t.Fatalf("expected %d rows, got %d", count, countAfter)
 	}
 }
+
+func TestPGWireTooManyArguments(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	s, _, _ := serverutils.StartServer(t, base.TestServerArgs{})
+	defer s.Stopper().Stop(context.TODO())
+
+	pgURL, cleanupFn := sqlutils.PGUrl(t, s.ServingAddr(), t.Name(), url.User(security.RootUser))
+	defer cleanupFn()
+
+	db, err := gosql.Open("postgres", pgURL.String())
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+
+	query := "SELECT"
+
+	args := make([]interface{}, 1<<16)
+	for i := 1; i <= 1<<16; i++ {
+		comma := ","
+		if i == 1 {
+			comma = ""
+		}
+		query += fmt.Sprintf("%s $%d::int", comma, i)
+		args[i-1] = i
+	}
+	query += ";"
+
+	if _, err := db.Prepare(query); !testutils.IsError(err, "more than 65535 arguments") {
+		t.Fatalf("unexpected error: %v", err)
+	}
+}

--- a/pkg/sql/pgwire/v3.go
+++ b/pkg/sql/pgwire/v3.go
@@ -36,6 +36,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/cockroach/pkg/util/tracing"
+	"math"
 )
 
 //go:generate stringer -type=clientMessageType
@@ -517,6 +518,13 @@ func (c *v3Conn) handleSimpleQuery(buf *readBuffer) error {
 	return c.finishExecute(results, nil, true, 0)
 }
 
+// maxPreparedStatementArgs is the maximum number of arguments a prepared
+// statement can have when prepared via the Postgres wire protocol. This is not
+// documented by Postgres, but is a consequence of the fact that a 16-bit
+// integer in the wire format is used to indicate the number of values to bind
+// during prepared statement execution.
+const maxPreparedStatementArgs int = math.MaxUint16
+
 func (c *v3Conn) handleParse(buf *readBuffer) error {
 	name, err := buf.getString()
 	if err != nil {
@@ -567,6 +575,10 @@ func (c *v3Conn) handleParse(buf *readBuffer) error {
 	}
 	// Convert the inferred SQL types back to an array of pgwire Oids.
 	inTypes := make([]oid.Oid, 0, len(stmt.SQLTypes))
+	if len(stmt.SQLTypes) > maxPreparedStatementArgs {
+		return c.sendError(pgerror.NewErrorf(pgerror.CodeProtocolViolationError,
+			"more than 65535 arguments to prepared statement: %d", len(stmt.SQLTypes)))
+	}
 	for k, t := range stmt.SQLTypes {
 		i, err := strconv.Atoi(k)
 		if err != nil || i < 1 {

